dev-docs/explanations/translations.md

```diff
@@ -0,0 +1,66 @@
+# Translations
+
+Ubuntu Pro Client is a special package in Ubuntu: it is updated frequently in all Ubuntu LTS releases.
+That makes language packs unsuitable for providing and updating translations for Ubuntu Pro Client.
+
+In addition, Ubuntu Pro Client includes lots of language with potential commercial and contractual
+implications related to Canonical's Ubuntu Pro offering. That means we need to maintain tight control
+over the language used in all messages.
+
+## How are translations delivered and updated?
+
+Normally, packages in `main` are translated via language packs; however language packs are only updated
+in stable releases around the time of point releases. If Ubuntu Pro translations were put in the language pack,
+then the language pack wouldn’t get updated as we continue to SRU new Ubuntu Pro Client versions back to
+old Ubuntu releases. That would cause the translations provided by the language pack to drift out of sync with
+the version of Ubuntu Pro Client installed.
+
+Because of this, Ubuntu Pro Client is not included in the Ubuntu language pack and instead ships all translations
+directly in our own source package. However, translations are kept in a separate binary package so that it is not
```

Comment:
"it is not required" - unclear what "it" refers to in this part

Reply:
Updating it to be clearer